istioctl/pkg/writer/envoy/configdump/testdata/secret/output

```diff
@@ -0,0 +1,3 @@
+RESOURCE NAME     TYPE           STATUS     VALID CERT     SERIAL NUMBER                               NOT AFTER                NOT BEFORE
+default           Cert Chain     ACTIVE     false          148535434475073015277422257846522681107     2023-05-16T01:32:52Z     2023-05-15T01:30:52Z
```

Comment:
The only thing I fixed was the cert `SERIAL NUMBER`, which is not related to the fix in this PR. The displayed serial number format was revised after release 1.17, so I merely reverted it to follow the previous behavior.